scan.py

```diff
@@ -221,6 +223,14 @@ def lambda_handler(event, context):
     if s3_object is None:
         return
 
+    try:
+        # If the size of the S3 object exceeds an allowed size, skip AV scanning
+        if s3_object.content_length >= ALLOWED_FILE_SIZE:
+            return
```

Comment:
Can you add a log in there? Is there some way to add the AV status tag as something like SKIP_LARGE_FILE?

Reply:
> Can you add a log in there?

Sure.

> Is there some way to add the AV status tag as something like SKIP_LARGE_FILE?
This makes sense doing this upon uploading a file to S3 where we have to check a file size or something and place the tag on the file. After that, we should check whether the tag exists or not before doing AV scanning.

The second option for placing tags is to do it here too https://github.com/solvebio/bucket-antivirus-function/pull/3/files/832fb1df0dd5acc76f1f159162b74946bd437538#diff-8d609b820a43e9e28fe7d05ae0f47c18a0aca37a6d3ab5be60194df0cd060be4R229

Do we want to use that tag somewhere else except only for the AV scanning?

Reply:
Great, I will handle it.